src/response.rs

```diff
@@ -193,3 +227,16 @@ pub fn parse_get_gj_acccount_comments_response(response: &str) -> Result<Vec<Pro
         .map(|fragment| Ok(ProfileComment::from_robtop_str(fragment)?))
         .collect()
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::response::parse_get_gj_comment_history_response;
+
+    #[test]
+    fn test_parse_comment_history_request() {
+        let test = parse_get_gj_comment_history_response("2~Rmlyc3Q=~1~76298358~3~3713125~4~2~10~0~9~1 week~6~33134786~11~2~12~75,255,75:1~Ryder~9~101~10~7~11~9~14~0~15~2~16~57903|2~TG92ZSB0byBzZWUgaXQ=~1~69201939~3~3713125~4~5~10~0~9~2 weeks~6~32301881~11~2~12~75,255,75:1~Ryder~9~101~10~7~11~9~14~0~15~2~16~57903#999:0:2");
+
+        println!("{:?}", test);
+    }
+
+}
```

Comment:
i don't think the tests are supposed to be like this, aren't they supposed to be in the tests folder and also... check that the function returned what it's supposed to

Reply:
Testing was there solely for me, typically we don't test on the response parsing functions, removed.